.github/workflows/cmake_build.yml

```
@@ -63,14 +77,34 @@ jobs:
     - name: Environment
       run: |
         source "scripts/ci/environment.sh"
-    
+
     - name: Install Apple codesigning certificates
       uses: apple-actions/import-codesign-certs@v1
       if: startswith( matrix.config.os, 'macos' ) && github.event_name == 'push' && github.repository_owner == 'audacity'
       with: 
         p12-file-base64: ${{ secrets.APPLE_CERTIFICATE }}
         p12-password: ${{ secrets.APPLE_CERTIFICATE_PASSWORD }}
 
+    # required for CMake to find Ninja
+    - name: "[Windows] Set up MSVC Developer Command Prompt"
+      if: runner.os == 'Windows'
+      uses: seanmiddleditch/gha-setup-vsdevenv@v3
+      with:
+        arch: ${{ matrix.config.arch }}
+
+    - name: "[Windows] Install sccache"
+      if: runner.os == 'Windows'
+      run: choco install sccache
+
+    - name: "Set up compiler cache"
+      uses: actions/cache@v2
+      with:
+        path: ${{ matrix.config.compiler_cache_path }}
+        key: ${{ matrix.config.os }}-${{ matrix.config.arch }}-${{ github.head_ref }}-${{ github.run_number }}
+        restore-keys: |
+          ${{ matrix.config.os }}-${{ matrix.config.arch }}-${{ github.head_ref }}
+          ${{ matrix.config.os }}-${{ matrix.config.arch }}
```

I know I've asked about that before but still... What's your opinion on making this a bit more clear?

```suggestion
          ${{ matrix.config.os }}-${{ matrix.config.arch }}-${{ github.head_ref }}-
          ${{ matrix.config.os }}-${{ matrix.config.arch }}-
```

---

This looks odd to me and I don't think it helps.

---

That does not particularly 'help' because the code is not broken (sorry again), but it makes it a bit more obvious that these lines denote prefixes, not exact keys. Also GitHub docs use trailing dashes in their examples too.

---

Okay, done. Caching is still working.